tests/integration.py

```diff
@@ -337,6 +337,8 @@ def unique_name(self, prefix):
         self.alerts_created_during_test.append(name)
         return name
 
+class TestSpaces(TestLibratoBase):
+
```

Comment:
Just realized I committed this - still working on integration testing. I'll fix.